house/views.py

```diff
@@ -365,7 +366,9 @@ def get_comments(request):
         }, status=404)
     comments_list = list()
     for each in query_set:
-        comments_list.append(each.comment)
+        one_single_comment = dict()
+        one_single_comment[each.user_email] = each.comment
+        comments_list.append(one_single_comment)
```

Comment:
Making `one_single_comment` be like `{ "email": "abc@bcd.com", "comment": "some content"}` would be easier for frontend to use.

Reply:
Sure, no problem